pennylane/circuit_drawer/charsets.py

```diff
@@ -0,0 +1,148 @@
+# Copyright 2020 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+This module contains the different charsets supported by PennyLane's CircuitDrawer.
+"""
+import abc
+
+
+class CharSet(abc.ABC):
+    """Charset base class."""
+
+    # pylint: disable=too-few-public-methods
+
+    WIRE = None
+    MEASUREMENT = None
+    TOP_MULTI_LINE_GATE_CONNECTOR = None
+    MIDDLE_MULTI_LINE_GATE_CONNECTOR = None
+    BOTTOM_MULTI_LINE_GATE_CONNECTOR = None
+    EMPTY_MULTI_LINE_GATE_CONNECTOR = None
+    CONTROL = None
+    LANGLE = None
+    RANGLE = None
+    VERTICAL_LINE = None
+    CROSSED_LINES = None
+    PIPE = None
+    OTIMES = None
```

Comment:
Small suggestion: in the abstract base class, you could also make the above abstract properties. This has two advantages:

1. inheriting classes are forced to overwrite them

2. the linter will then force you to add docstrings so that developers know where they are used :slightly_smiling_face: 

e.g., 

```python
@property
@abstractmethod
def WIRE(self):
    """The string to use to represent a wire"""
```

Reply:
I implemented it that way!